deploy/kubernetes/examples/demo-secret-configuration-file.json

```diff
@@ -0,0 +1,24 @@
+{
+  "demo-system_id1": {

```

Comment:
I know `by_system_id` has underscores (according to our controller convention), but I think our examples names convention should remain with hyphens, i.e. `demo-system-id1`

Reply:
I don't mind having `_`/`-`, but I suggest not mixing them together in one string. In this case, `demo_system_id1` and `demo-system-id1` are ok IMO

Reply:
ok, done

Reply:
agree with @zingero  just be sure to keep the hyphenation/underscores to be consistent as possible - especially within the same files

Reply:
it is, keys and values were always different and I don't think we should change that..

Reply:
could you give an example of an existing "mixed-convention" value?

Reply:
within the same file?
in storageclass:
keys:
csi.storage.k8s.io/controller-expand-secret-name
volume_name_prefix
values:
demo-pool

Reply:
no, not same file, same value.
there is a **key** difference between key and value.

regarding `demoPVC` - IMO it should be something like `demo-prefix`.

Reply:
what is the AI?